views/index.jsx

```diff
@@ -25,7 +25,7 @@ function Modal() {
     <section className="modal modal--hidden">
       <div className="modal__content">
         <div className="modal__video">
-          <iframe src="https://www.youtube.com/embed/QH2-TGUlwu4" width="560" height="315" frameborder="0"></iframe>
+          <iframe src="https://www.youtube.com/embed/QH2-TGUlwu4" width="560" height="315" frameBorder="0"></iframe>
```

Comment:
What is the difference between `frameborder` and `frameBorder`? I think HTML is case-insensitive